metaflow/plugins/pypi/conda_decorator.py

```diff
@@ -173,7 +173,10 @@ def runtime_init(self, flow, graph, package, run_id):
             # EXT_PKG extensions are PYTHONPATH extensions. Instead of re-resolving,
             # we use the resolved information that is written out to the INFO file.
             with open(
-                os.path.join(self.metaflow_dir.name, os.path.basename(INFO_FILE)),
+                os.path.join(
+                    self.metaflow_dir.name,
+                    os.path.basename(MetaFile.INFO_FILE.value),
```

Comment:
`MetaFile.INFO_FILE.value` is just a filename so `os.path.basename` here seems unnecessary